patches/gcc/linaro-6.3-2017.02/952-bionic-errno.patch

```diff
@@ -0,0 +1,19 @@
+commit 6cd4ad106ef87a3c58b0c3478e78409b47000de0
```

Comment:
No, I believe the same patches would be needed for stock 6.3. I hadn't tried it yet, but will try it now.